src/code/z_parameter.c

```diff
@@ -94,13 +1050,96 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80115844.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80115908.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_801159c0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_801159EC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80115A14.s")
+/**
+ * Returns true if player still has health left. Otherwise, return false.
```

Comment:
```suggestion
 * @return boolean true if player still has health left, else false.
```
I think this would be just as clear.